map/src/contextmenu/LocalTrackTabList.js

```diff
@@ -0,0 +1,29 @@
+import LocalInfoTab from "./components/tabs/LocalInfoTab";
```

Comment:
Should be definitely combined with TrackTabList in future there is no reason that ElevationTab or SpeedTab won't be accessible for local tracks, so it's non future proof.